src/Ledger/Conway/Conformance/Epoch.agda

```diff
@@ -170,14 +172,14 @@ data _⊢_⇀⦇_,EPOCH⦈_ : ⊤ → EpochState → Epoch → EpochState → Ty
 
       certState' =
         ⟦ record dState { rewards = dState .rewards ∪⁺ refunds }
-        , ⟦ (pState .pools) ∣ retired ᶜ , (pState .retiring) ∣ retired ᶜ ⟧ᵖ
+        , ⟦ (pState .pools) ∣ retired ᶜ , (pState .retiring) ∣ retired ᶜ ⟧
         , ⟦ if null govSt' then mapValues (1 +_) (gState .dreps) else (gState .dreps)
           , (gState .ccHotKeys) ∣ ccCreds (es .cc)
           , vDeposits
           ⟧ᵛ
```

Comment:
Should you also remove superscripts on constructors in the conformance version of the `Epoch` module?